client/src/app/site/pages/meetings/pages/motions/services/export/motion-pdf-export.service/motion-pdf-export.service.ts

```diff
@@ -52,6 +59,48 @@ export class MotionPdfExportService {
         this.pdfDocumentService.download({ docDefinition, filename, metadata, exportInfo });
     }
 
+    /**
+     * Exports multiple motions to a collection of PDFs including pdf attachments
+     *
+     * @param motions the motions to export
+     */
+    public async exportMotionCatalogWithAttachments(
```

Comment:
does this need to be public?
```suggestion
    private async exportMotionCatalogWithAttachments(
```